OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -46,10 +49,8 @@ public class ArmamentInfo : UpgradableTraitInfo, ITraitInfo, Requires<AttackBase
 
 		[Desc("Muzzle flash sequence to render")]
 		public readonly string MuzzleSequence = null;
-
```

Comment:
Don't mix coding style fixes/clarity (if they don't change logic) with logical changes.
If you are going to do this create a new commit.